src/main/java/org/commcare/formplayer/screens/FormplayerQueryScreen.java

```diff
@@ -15,19 +10,4 @@
     public FormplayerQueryScreen(){
         super(null, null, null);
     }
-
-    /**
-     *
-     * @param skipDefaultPromptValues don't apply the default value expressions for query prompts
-     * @return case search url with search prompt values
-     */
-    public URI getUri(boolean skipDefaultPromptValues) {
```

Comment:
Removed in favour of [QueryScreen.buildUrl](https://github.com/dimagi/commcare-core/blob/master/src/cli/java/org/commcare/util/screen/QueryScreen.java#L94)

Reply:
Had to add it back since QueryScreen.buildURI in core encodes the url params which results in double encoding url params on Foromplayer since `restTemplate`  by default encodes the url in Spring.

Reply:
Pity about the duplication - we could create a separate RestTemplate instance that doesn't do the encoding or possibly some other workaround.

Reply:
yeah I am not sure if it's worth creating another RestTemplate instance just to avoid duplication here. It feels like we will be introducing complexity to avoid complexity. I am going to leave it as it is in context of this PR and can revisit if it bothers us again.